src/protocols/bitcoin/views/AccountDetailsTransactions.vue

```diff
@@ -101,7 +96,7 @@ export default defineComponent({
 
     return {
       isOnline,
-      transactions,
+      transactions: transactionsLoaded,
```

Comment:
Instead of renaming the returned const name please update the template to use `transactionsLoaded`.